index.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path")
+const express = require("express")
+const uploadRoute = require('./controller/routeUpload')
+const extractTextRoute = require('./controller/extractTextRoute')
+const upload = require("./middleware/multer");
```

Comment:
Are we using multer for storing files in the application? If not you can remove this export

Reply:
Removed.